docs/src/rules/class-methods-use-this.md

```diff
@@ -6,55 +6,91 @@ further_reading:
 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static
 ---
 
+[Classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) are often used to encapsulate reusable logic, especially stateful logic, into an object, where each instance's state is accessed via `this`. When an API is written with an instance method, it signals to consumers:
```

Comment:
[Readability] This is a lot of commas. I had a hard time keeping track of which clause applied to what.

```suggestion
[Classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) are often used to encapsulate reusable logic, especially stateful logic, into an object where each instance's state is accessed via `this`. When an API is written with an instance method, it signals to consumers:
```